ingestion/src/metadata/ingestion/ometa/mixins/data_insight_mixin.py

```diff
@@ -175,13 +175,15 @@ def delete_web_analytic_event_before_ts_exclusive(
         event_type_value = event_type.value
         self.client.delete(f"/analytics/web/events/{event_type_value}/{tmsp}/collect")
 
-    def delete_report_data(self, report_data_type: ReportDataType, date: str) -> None:
+    def delete_report_data(
```

Comment:
let's create a separate method `delete_report_data_at_date` and `delete_report_data`. We don't want to mistakenly delete everything because we forgot to pass the date parameter